website/api/controllers/deliver-apple-csr.js

```diff
@@ -113,34 +120,46 @@ module.exports = {
       organization: generateCertificateResult.org,
     });
 
-    // Send an email to the user, with the result from the mdm-gen-cert command attached as a plain text file.
-    await sails.helpers.sendTemplateEmail.with({
-      to: generateCertificateResult.email,
-      subject: 'Your certificate signing request from Fleet',
-      from: sails.config.custom.fromEmailAddress,
-      fromName: sails.config.custom.fromName,
-      template: 'email-signed-csr-for-apns',
-      templateData: {},
-      attachments: [{
-        // When the file is provided as an attachment to the Sails helper, it
-        // gets decoded, since we need for the signed CSR to be delivered in
-        // base64 format, we doubly encode the contents before sending the
-        // email.
-        contentBytes: Buffer.from(generateCertificateResult.request).toString('base64'),
-        name: 'apple-apns-csr.txt',
-        type: 'text/plain',
-      }],
-    }).intercept((err)=>{
-      return new Error(`When trying to send a signed CSR to a user (${generateCertificateResult.email}), an error occured. Full error: ${err}`);
-    });
+    // respBody contains the raw response body content, it defaults to
+    // `undefined` as this was the default behavior of this endpoint before the
+    // 'alt' parameter was introduced.
+    var respBody;
+    switch(alt) {
+      case 'json':
+        this.res.type('application/json');
+        respBody = JSON.stringify({
+          csr: Buffer.from(generateCertificateResult.request).toString('base64'),
+        });
+      default:
```

Comment:
Did you mean to fall through here? JS requires `break` or `return` otherwise it falls through (unlike Go). If this is intentional, it would probably be good to add a comment (or just use an `if` block).

Reply:
thank you! added a `break`